src/Microsoft.Docs.MarkdigExtensions/ResolveLink/ResolveLinkExtension.cs

```diff
@@ -47,6 +47,33 @@ private void UpdateLinks(MarkdownObject markdownObject)
                 autolinkInline.Url = _context.GetLink(autolinkInline.Url, autolinkInline);
                 break;
 
+            case QuoteSectionNoteBlock quoteSectionNoteBlock:
+                if (quoteSectionNoteBlock.QuoteType == QuoteSectionNoteType.DFMVideo)
+                {
+                    quoteSectionNoteBlock.VideoLink = _context.GetLink(quoteSectionNoteBlock.VideoLink, quoteSectionNoteBlock);
+                }
+                break;
+
+            case TripleColonBlock tripleColonBlock:
```

Comment:
We are removing `VideoExtension` so likely don't need this. To test, we usually write an e2e spec test, see [markdown.yml](https://github.com/dotnet/docfx/blob/v3/docs/specs/markdown.yml#L790) for examples.